Purchases/Purchasing/RCStoreKitWrapper.m

```diff
@@ -75,28 +70,48 @@ - (void)presentCodeRedemptionSheet API_AVAILABLE(ios(14.0)) API_UNAVAILABLE(tvos
     }
 }
 
-- (void)paymentQueue:(SKPaymentQueue *)queue updatedTransactions:(NSArray<SKPaymentTransaction *> *)transactions
-{
+- (void)paymentQueue:(SKPaymentQueue *)queue
+ updatedTransactions:(NSArray<SKPaymentTransaction *> *)transactions {
     for (SKPaymentTransaction *transaction in transactions) {
-        RCDebugLog(@"PaymentQueue updatedTransaction: %@ %@ (%@) %@ - %ld", transaction.payment.productIdentifier, transaction.transactionIdentifier, transaction.error, transaction.originalTransaction.transactionIdentifier, (long)transaction.transactionState);
+        RCDebugLog(@"PaymentQueue updatedTransaction: %@ %@ (%@) %@ - %ld",
+                   transaction.payment.productIdentifier,
+                   transaction.transactionIdentifier,
+                   transaction.error,
+                   transaction.originalTransaction.transactionIdentifier,
+                   (long)transaction.transactionState);
         [self.delegate storeKitWrapper:self updatedTransaction:transaction];
     }
 }
 
 // Sent when transactions are removed from the queue (via finishTransaction:).
-- (void)paymentQueue:(SKPaymentQueue *)queue removedTransactions:(NSArray<SKPaymentTransaction *> *)transactions
-{
+- (void)paymentQueue:(SKPaymentQueue *)queue
+ removedTransactions:(NSArray<SKPaymentTransaction *> *)transactions {
     for (SKPaymentTransaction *transaction in transactions) {
-        RCDebugLog(@"PaymentQueue removedTransaction: %@ %@ (%@ %@) %@ - %ld", transaction.payment.productIdentifier, transaction.transactionIdentifier, transaction.originalTransaction.transactionIdentifier, transaction.error, transaction.error.userInfo, (long)transaction.transactionState);
+        RCDebugLog(@"PaymentQueue removedTransaction: %@ %@ (%@ %@) %@ - %ld",
+                   transaction.payment.productIdentifier,
+                   transaction.transactionIdentifier,
+                   transaction.originalTransaction.transactionIdentifier,
+                   transaction.error,
+                   transaction.error.userInfo,
+                   (long)transaction.transactionState);
         [self.delegate storeKitWrapper:self removedTransaction:transaction];
     }
 }
 
 #if PURCHASES_INITIATED_FROM_APP_STORE_AVAILABLE
-- (BOOL)paymentQueue:(SKPaymentQueue *)queue shouldAddStorePayment:(SKPayment *)payment forProduct:(SKProduct *)product
-{
+- (BOOL)paymentQueue:(SKPaymentQueue *)queue
+shouldAddStorePayment:(SKPayment *)payment
+          forProduct:(SKProduct *)product {
```

Comment:
super minor, but looks like the `:` are not vertically aligned

Reply:
yeah, this is xcode's default alignment if the second line's argument name is longer than the first one's... I think AppCode did something better where it adds empty space, I'll try that

Reply:
@vegaro updated